ci/docker/common.yaml

```diff
@@ -16,16 +16,19 @@ packages:
       - '~cuda'
       - '~openmp'
       - '~rocm'
+  fmt:
+    version:
+      - 9.1.0
```

Comment:
What about adding your comment here for "future we" so that we can easily recall why it's there and eventually remove it?

Reply:
Very good point. I should actually just add a conflict to the spack package so that we can remove this constraint altogether. I'll update this PR once the constraint is in spack develop.

Reply:
See https://github.com/spack/spack/pull/37679.

Reply:
Updated in https://github.com/eth-cscs/DLA-Future/pull/859/commits/3f0c34dc5f40038a1ed5123acea044d4eda38628.